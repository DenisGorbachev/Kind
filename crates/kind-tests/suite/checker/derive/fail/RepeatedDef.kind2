type Data.Nat {
    zero
    succ
}

Data.Nat.zero : Data.U60
Data.Nat.zero = 2