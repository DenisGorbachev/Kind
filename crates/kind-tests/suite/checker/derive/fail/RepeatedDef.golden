   ERROR  Defined multiple times for the same name

      /--[suite/checker/derive/fail/RepeatedDef.kind2:2:5]
      |
    1 |    type Data.Nat {
    2 |        zero
      |        v---
      |        \The first ocorrence
    3 |        succ
      :     
    6 |    Data.Nat.zero : Data.U60
      |    v------------
      |    \Second occorrence here!

      Hint: Rename one of the definitions or remove and look at how names work in Kind at https://github.com/Kindelia/Kind2/blob/master/guide/naming.md

